src/common/inject/inject.jsx

```diff
@@ -203,15 +203,54 @@ Zotero.Inject = new function() {
 		return deferred.promise;	
 	};
 	
+	this.firstSaveToServerPrompt = function() {
+		return this.confirm({
+			checkbox: true,
+			checkboxBlock: true,
+			checkboxText: "I want to save directly to zotero.org",
+			button3Text: "More Info...",
+			title: `Saving to zotero.org`,
+			message: `
+				Zotero Connectors allow saving directly to your zotero.org account when Zotero is not open.<br/><br/>
+				The preferred way to save items and manage your library is by using Zotero on your computer.
+			`
+		}).then(function(result) {
+			if (result.button == 3) {
+				Zotero.Connector_Browser.openTab('https://docs.google.com/document/d/1HNWG6pjjoyb9TfTvNj15KyxDiVLJQ_eWj9PR8fCgDXk');
```

Comment:
https://www.zotero.org/support/save_to_zotero_org

(I have these Docs URLs saved and will add content to these pages.)